cloudant-sync-datastore-core/src/main/java/com/cloudant/mazha/CouchClient.java

```diff
@@ -406,7 +405,7 @@ public void putAttachmentStream(String id, String rev, String attachmentName, St
                                                       Collection<String> attsSince,
                                                       boolean pullAttachmentsInline) {
         Misc.checkNotNullOrEmpty(id, "id");
-        Preconditions.checkArgument(revisions.size() > 0, "Need at lease one open revision");
+        Misc.checkArgument(revisions.size() > 0, "Need at lease one open revision");
```

Comment:
Many of the changes were made by regex, so I didn't see the error messages - nice spot! Fixed in 122e842.